### PR TITLE
Testbench: test image from file

### PR DESCRIPTION
**Pull request by ktovletoglou**
Instead of the static image for the testbench, you can choose an image from a file.
img2txt.m: taken from the original mkjpeg project, converts a bmp to a txt for the testbench

I did this pull request to see how git works, how to fork and how to manage the repo.

### CODE REVIEW

File: release/R1/sim/License.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Requested permission for usage of the images in the testing of the jpeg_encoder
```
Comment (shenki) [inline release/R1/sim/License.txt:1]: What do you mean here?

What is the license for the code you have copied from mkjpeg project?

File: release/R1/sim/License.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+Copyright, Tovletoglou Apostolos,2013
+https://plus.google.com/+ApostolosTovletoglou
```
Comment (shenki) [inline release/R1/sim/License.txt:3]: I suggest using you email address not your social media profiles.

**Comment by shenki** [comment]
Hello. I am a mentor from the timvideos GSoC project. I tried to test your patch but I can't seem to find a repository for me to clone. Can you please make one available?

File: release/R1/sim/License.txt (hunk 1 above)
Comment (ktovletoglou) [inline release/R1/sim/License.txt:1]: This is for the image. The license of the files, I copied from the mkjpeg is under the same license as the mkjpeg project so as already the license of the mkjpeg project is respected, they is nothing to do for these parts

**Comment by ktovletoglou** [comment]
Sorry I already deleted my fork of the HDMI2USB. I cannot supply you with a new repo as I am away from my main workstation for the next week.

**Comment by shenki** [comment]
Without the code, I am unable to test your submission.

**Comment by mithro** [comment]
The [HDMI2USB-jahanzeb-firmware](https://github.com/timvideos/HDMI2USB-jahanzeb-firmware) is no longer under development. 

All active development is now on the [HDMI2USB-misoc-firmware](https://github.com/timvideos/HDMI2USB-misoc-firmware) which supports both the [Atlys](https://hdmi2usb.tv/digilent-atlys/) and [Opsis](https://hdmi2usb.tv/numato-opsis/) boards and has loads of other new features!

**Pull request closed without merging**